### PR TITLE
Add limit pvp rewards

### PR DESCRIPTION
**Pull request by nicoaravena**
This PR add an extra option for the PVP rewards, allowing to limit the given rewards to an specific amount of characters in the top of the resultant value.

Test todo:
- Give rewards with or without the limit.
- Give rewards with an extra top rewards.

### CODE REVIEW

File: srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php [hunk 1]
```diff
@@ -505,15 +505,15 @@ public function getPvpRewardsRender($amount, $isWinner, $bracket, $month, $year,
...
+                                                    <option value=1 <?php if ($data['bracket'] == 1) echo 'selected'; ?>>10-19</option>
+                                                    <option value=2 <?php if ($data['bracket'] == 2) echo 'selected'; ?>>20-29</option>
+                                                    <option value=3 <?php if ($data['bracket'] == 3) echo 'selected'; ?>>30-39</option>
+                                                    <option value=4 <?php if ($data['bracket'] == 4) echo 'selected'; ?>>40-49</option>
+                                                    <option value=5 <?php if ($data['bracket'] == 5) echo 'selected'; ?>>50-59</option>
+                                                    <option value=6 <?php if ($data['bracket'] == 6) echo 'selected'; ?>>60-69</option>
+                                                    <option value=7 <?php if ($data['bracket'] == 7) echo 'selected'; ?>>70-79</option>
+                                                    <option value=8 <?php if ($data['bracket'] == 8) echo 'selected'; ?>>80</option>
```
Comment (Helias) [inline srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php:516]: :S, I suggest to use a for cycle here, like:
```php
<?php
foreach (brackets as $key, bracket) {
?>
<option value="<?= $key ?>" <?php if $data['bracket'] == $key) { echo 'selected'; } ?> ....bracket... </option>
<?php
}
?>
```

something similar, if you have no time, let's refactor this later

File: srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php [hunk 2]
```diff
@@ -678,6 +684,13 @@ public function getPvpRewardsRender($amount, $isWinner, $bracket, $month, $year,
                 var r = confirm("You sure you want to continue?");
                 return r;
```
Comment (Helias) [inline srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php:685]: you can even return confirm() directly

never use `var`, use only `let` or `const`, in this case, but even below I suggest to use const, but checking the code below `tooltipList` is not used, you can even don't declare that variable probably and you don't need to use .map() but .forEach()

Keep in mind that .map() return an array that "should be" assigned to a variable, while .forEach() doesn't.

Anyways, these are minor things, if you have no time, I am totally fine with this

File: srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php (hunk 1 above)
Comment (nicoaravena) [reply srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php]: Done!

File: srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php (hunk 2 above)
Comment (nicoaravena) [reply srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php]: Done!

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
I made standard test to be sure that the shop and user registration is ok, I would like to test also the pvp stats rewards system but locally I have not enough data to test this, I'll try later.

For what I tested so far, this PR is safe and we can merge it

**Pull request merged**